src/v2/guide/index.md

```diff
@@ -47,58 +47,58 @@ var app = new Vue({
 var app = new Vue({
   el: '#app',
   data: {
-    message: 'Hello Vue!'
+    message: 'Hello Vue !'
   }
 })
 </script>
 {% endraw %}
 
-We have already created our very first Vue app! This looks pretty similar to just rendering a string template, but Vue has done a lot of work under the hood. The data and the DOM are now linked, and everything is now **reactive**. How do we know? Just open your browser's JavaScript console (right now, on this page) and set `app.message` to a different value. You should see the rendered example above update accordingly.
+Nous venons tout juste de créer notre première application Vue ! Ça ressemble sacrément à du simple rendu dans un template, mais Vue effectue un réel travail dans l'ombre. Les données et le DOM sont maintenant couplés, et tout est à présent **réactif**. Comment s'en rendre compte ? Ouvrez juste la console JavaScript de votre navigateur (là maintenant, sur cette page) et attribuez à `app.message` différentes valeurs. Vous devriez voir le rendu de l'exemple en cours se mettre à jour en concordance.
```

Comment:
est-ce que cette notion de "magie fantastique" ne viendrait-elle pas plutôt de ton imagination ? :smile:
"sous le capot" est la traduction littérale de "under the hood", les deux expressions sont les mêmes en anglais comme en français